src/main/java/racingcar/service/CarService.java

```diff
@@ -0,0 +1,75 @@
+package racingcar.service;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import racingcar.domain.Car;
+
+public class CarService {
+
+    private static final String NAME_LENGTH_ERROR_MESSEAGE = "[ERROR] 자동차의 이름은 1글자 이상 5글자 이하여야 합니다.";
+    private static final String NUMBER_RANGE_ERROR_MESSEAGE = "[ERROR] 시행 횟수는 1 이상 2,147,483,647 이하의 숫자만 가능합니다.";
+    public static final int MAXIMUM_NAME_LENGTH = 5;
+    public static final int MINIMUM_NAME_LENGTH = 1;
+
+    public String[] splitCarNames(final String names) {
+        return names.split(",");
+    }
+
+    public List<Car> race(final List<Car> cars) {
+        Random random = new Random();
+        return cars.stream()
+                .map(c -> c.race(random.nextInt(10)))
+                .collect(Collectors.toList());
+    }
+
+    public List<String> getWinners(final List<Car> cars) {
+        final int max = getMaxPosition(cars);
+        return cars.stream()
+                .filter(c -> c.getPosition() == max)
+                .map(c -> c.getName())
+                .collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    public List<String> getWinnerNames(final List<Car> cars) {
        final int max = getMaxPosition(cars);
        return cars.stream()
                .filter(c -> c.getPosition() == max)
                .map(c -> c.getName())
                .collect(Collectors.toList());
    }
```
메서드명을 더 상세하게 적는게 좋을 것 같네요.